src/sentry/static/sentry/app/utils/streamManager.jsx

```diff
@@ -24,6 +24,8 @@ class StreamManager {
     items = [].concat(items);
     if (items.length === 0) return this;
 
+    items = items.filter((item) => item.hasOwnProperty("id"));
```

Comment:
We didn't see any real instances of it but we just noticed it was possible during testing.